### ISSUE TITLE
Error: Not Found

### ISSUE THREAD

**Issue by gtamang001** [imranismail/setup-kustomize#282]
I am trying to implement this action in my Organization. We run workflows against our own runners, mostly centos vms. Once added this action fails with "Error: Not Found." No additional description is available. Is it something you can look into ?

**Comment by imranismail** [comment]
can you share the inputs you passed?

**Comment by gtamang001** [comment]
I tried few combinations and permutations. I tried without any inputs as well. With inputs I used the as below, 
with: 
     github-token: {{github.token}}
     kustomize-version: ( some variations of these) 10.15.1, 4.5.7, *  etc
     fail-fast: true
I have tried
FYI I tested with no inputs on my PC with ububutu-latest and it works, the only difference is I am running against runners setup in my organization.

**Comment by hkuersten** [comment]
I have the same issue.
![Screenshot 2023-08-29 at 09 51 59](https://github.com/imranismail/setup-kustomize/assets/22233344/9cb672f7-9aaf-4db2-8d30-e28a7ace5dc9)

**Comment by hkuersten** [comment]
Could this be a Firewall issue?? We have very restrictive Firewall settings in our cluster, need to whitelist pretty much everything. From which location will this action be downloaded?

**Comment by hkuersten** [comment]
Disclaimer: I am not a very proficient coder, but could it be that the method _getMaxSatisfyingVersion_ in file [installer.ts](https://github.com/imranismail/setup-kustomize/blob/master/src/installer.ts) is not finding a suitable version because of the _versionRegex_ ?
That is set to `/\d+\.?\d*\.?\d*/` but if I correctly read the results returned by the Github API, the field `tag_name` will not match this regular expression:
```
# gh api repos/kubernetes-sigs/kustomize/releases
[
  {
    "url": "https://api.github.com/repos/kubernetes-sigs/kustomize/releases/114535131",
    "assets_url": "https://api.github.com/repos/kubernetes-sigs/kustomize/releases/114535131/assets",
    "upload_url": "https://uploads.github.com/repos/kubernetes-sigs/kustomize/releases/114535131/assets{?name,label}",
    "html_url": "https://github.com/kubernetes-sigs/kustomize/releases/tag/kustomize/v5.1.1",
    "id": 114535131,
    "author": {
      "login": "KnVerey",
      "id": 4789493,
      "node_id": "MDQ6VXNlcjQ3ODk0OTM=",
      "avatar_url": "https://avatars.githubusercontent.com/u/4789493?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/KnVerey",
      "html_url": "https://github.com/KnVerey",
      "followers_url": "https://api.github.com/users/KnVerey/followers",
      "following_url": "https://api.github.com/users/KnVerey/following{/other_user}",
      "gists_url": "https://api.github.com/users/KnVerey/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/KnVerey/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/KnVerey/subscriptions",
      "organizations_url": "https://api.github.com/users/KnVerey/orgs",
      "repos_url": "https://api.github.com/users/KnVerey/repos",
      "events_url": "https://api.github.com/users/KnVerey/events{/privacy}",
      "received_events_url": "https://api.github.com/users/KnVerey/received_events",
      "type": "User",
      "site_admin": false
    },
    "node_id": "RE_kwDOB-5y6s4G06rb",
    "tag_name": "kustomize/v5.1.1",
    "target_commitish": "master",
    "name": "kustomize/v5.1.1",
    "draft": false,
    "prerelease": false,
    "created_at": "2023-07-31T17:10:47Z",
    "published_at": "2023-07-31T17:20:01Z",
    (...)
```
Note the value is **kustomize/v5.1.1**. Maybe this is the problem?

**Comment by imranismail** [comment]
I'll investigate this, it could be that they've changed the naming convention again.

**Comment by imranismail** [comment]
I've released v2.1.0-rc which adds significant speed bump for pinned version lookup. I couldn't reproduce this issue after the change: https://github.com/imranismail/setup-kustomize/actions/runs/6622780097/job/17988783976
